PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -371,16 +392,35 @@ class Driver {
     }
 
     async initialize() {
+        await this.preloadFilesForBrowser();
         await this.fetchResources();
         this.prepareToRun();
         if (isInBrowser && window.location.search == '?report=true') {
             setTimeout(() => this.start(), 4000);
         }
     }
 
+    async preloadFilesForBrowser() {
+        if (!isInBrowser)
+            return;
+
+        var counter = { };
+        counter.loadedFiles = 0;
+        counter.totalFiles = 0;
+
+        var promises = [];
+        for (let benchmark of this.benchmarks)
+            promises.push(benchmark.preloadFilesForBrowser(counter));
+        await Promise.all(promises);
+
+        window.loadCache = undefined; // Done preloading all the files.
```

Comment:
Let's make it `this.loadCache`. And let's clear it with `this.loadCache = {};`. So we always have an object.
And let's put `this.loadCache = {};` in the constructor of this `Driver`.